packages/protocol/contracts/governance/Election.sol

```diff
@@ -790,15 +798,91 @@ contract Election is
   }
 
   /**
-   * @notice Returns a list of elected validators with seats allocated to groups via the D'Hondt
+   * @notice Returns a list of elected validator signers with seats allocated to groups via the D'Hondt
    *   method.
-   * @return The list of elected validators.
+   * @return The list of elected validator signers.
    * @dev See https://en.wikipedia.org/wiki/D%27Hondt_method#Allocation for more information.
    */
   function electNValidatorSigners(
     uint256 minElectableValidators,
     uint256 maxElectableValidators
-  ) public view returns (address[] memory) {
+  ) public view onlyL1 returns (address[] memory) {
```

Comment:
Since this function is identical (besides modifier) to `electNValidatorAccounts` does it make sense to have it ? Could we remove just the modifier and keep one function called eg `electNValidators` instead ?